### PR TITLE
rh7: Suppress mlx5 ib driver unknown symbol warning during LIS installation

### PR DESCRIPTION
**Pull request by johnsongeorge-w**

### CODE REVIEW

**Comment by johnsongeorge-w** [comment]
@prashantmhetar @santoshx Review the modification.

File: LISISO/commonfunctions.sh [hunk 1]
```diff
@@ -247,8 +255,9 @@ function installbuildrpm()
 	if [ "$kmodrpm" != "" ] && [ "$msrpm" != ""  ];
 	then
 		echo "Installing the Linux Integration Services for Microsoft Hyper-V..."
-		rpm -ivh $kmodrpm $msrpm
+		rpm -ivh $kmodrpm $msrpm 2> ${ERR_LOG_FILE}
```
Comment (santoshx) [inline LISISO/commonfunctions.sh]: Can't we filter out the warnings right here?
Something like rpm -ivh $kmodrpm $msrpm | grep -v "mlx5_ib".
Not understood why need to send to file and then cat the file.

**Review by santoshx — CHANGES_REQUESTED** [review]

File: LISISO/commonfunctions.sh (hunk 1 above)
Comment (johnsongeorge-w) [reply LISISO/commonfunctions.sh]: That will filter out the message appearing on stdout. Warning are thrown in stderr (fd:2). Hence capturing stderr and then filtering it out.

Comment (santoshx) [reply LISISO/commonfunctions.sh]: Did you try this?
rpm -ivh $kmodrpm $msrpm 2>&1 | grep -v mlx5_ib

Comment (johnsongeorge-w) [reply LISISO/commonfunctions.sh]: This should work. Re-directing stderr to stdout and filtering out.

**Review by santoshx — APPROVED** [review]

**Pull request merged**